homeassistant/components/postnl/sensor.py

```diff
@@ -90,6 +90,8 @@ def icon(self):
     @Throttle(MIN_TIME_BETWEEN_UPDATES)
     def update(self):
         """Update device state."""
-        shipments = self._api.get_relevant_shipments()
-        self._attributes["shipments"] = shipments
+        shipments = self._api.get_relevant_deliveries()
+        for shipment in shipments:
+            self._attributes["shipments"].append(vars(shipment))
```

Comment:
The `"shipments"` list will grow indefinitely with this code. That looks like a bug.

Reply:
Well spotted. Fixed!